containers/base/Dockerfile

```diff
@@ -257,8 +257,11 @@ RUN if [ -d /datalab/lib/pydatalab/.git ]; then \
     pip install --upgrade-strategy only-if-needed --no-cache-dir . && \
     pip install --upgrade-strategy only-if-needed /datalab/lib/pydatalab/solutionbox/image_classification/. && \
     pip install --upgrade-strategy only-if-needed /datalab/lib/pydatalab/solutionbox/structured_data/. && \
+    pip install --upgrade-strategy only-if-needed jupyter_highlight_selected_word && \
```

Comment:
This can apparently be installed with conda, but since we are only installing it in one environment I don't know if that provides any benefit.

I.E. I'm not asking you to change that; just noting that it is an option.